ecs-init/cache/cache.go

```diff
@@ -76,11 +91,34 @@ func NewDownloader() (*Downloader, error) {
 	return downloader, nil
 }
 
+func (d *Downloader) AgentCacheStatus() CacheStatus {
```

Comment:
It'd be good to add documentation for this method as well. More for understanding and for reference, it'd also be really useful to add a blurb about which cache status is returned when. For example, it's not obvious to me when `StatusReloadNeeded` will be returned from this method (unless someone happens to be looking at the rpm spec file).

Reply:
Agreed. I'll add some docs.